src/Promenade.Web/Views/Home/Locations.cshtml

```diff
@@ -0,0 +1,160 @@
+﻿@model Ocuda.Promenade.Models.Entities.Location
+@{
+    ViewData["Title"] = "Locations";
+}
+    <body>
+        <h1>Find A Library</h1>
+        <h2>Find your closest Maricopa County Library by entering your address or zip code:</h2>
+        <div class="body-content">
+
+            <div class="row">
+                <div class="col-xs-12" style="padding-bottom: 1rem;">
+                    <form asp-action="Index" method="get" role="form" id="AddressForm">
+
+                        <div class="input-group">
+                            <input asp-for="Address"
+                                   placeholder="Enter your address or zip code"
+                                   class="form-control default-text"
+                                   id="LocatorAddress" />
+                            <span class="input-group-btn">
+                                <button href="/LocateZip#" type="submit" class="btn btn-primary" id="GoButton">Go</button>
+                            </span>
+                        </div>
+                    </form>
+
+                </div>
+                    <div class="col-xs-12 @if (Model?.ShowLocation != true) {<text>hidden</text> }" style="padding-bottom: 1rem;">
+                         <form asp-action="Location" method="get" role="form" id="GeoForm">
+                             <input asp-for="Latitude" id="Latitude" hidden />
+                             <input asp-for="Longitude" id="Longitude" hidden />
+                         </form>
+                         <button type="button"
```

Comment:
This button isn't in the `<form>`?

Reply:
The form is only for submitting the address, This button just submits the current location